python/letsql/common/utils/dask_normalize_expr.py

```diff
@@ -196,6 +202,11 @@ def normalize_module(module):
     )
 
 
+@dask.base.normalize_token.register(dat.DataType)
+def normalize_ibis_datatype(datatype):
+    return dask.tokenize._normalize_seq_func((datatype.nullable, datatype.name.lower()))
```

Comment:
`DataType` can be complex, like `Struct`
```In [1]: import ibis; ibis.expr.datatypes.core.Struct({"a": ibis.expr.datatypes.int16}).args
Out[1]: ({'a': Int16(nullable=True)}, True)
```
I think here we should use
```
+@dask.base.normalize_token.register(dt.DataType)
+def normalize_ibis_datatype(obj):
+    return dask.tokenize._normalize_seq_func(obj.args)
```

Reply:
makes sense